inc/inventory.class.php

```diff
@@ -283,8 +283,7 @@ function computersByAge($config = array()) {
            AND itemtype = 'Computer'
             AND i.`warranty_date` IS NULL
             $sql_entities
-            $sql_states
-         ORDER BY Total DESC";
+            $sql_states";
```

Comment:
Thank's for approve :)

Yes I agree, for that we have to re-write the sql queries from above (this is not the purpose of this quick "fix").

I make an issue right now.